app/components/organization_component.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class OrganizationComponent < ApplicationComponent
+  def initialize(organization:, **)
+    @organization = organization
+
+    super
+  end
+
+  attr_reader :organization
+end
```

Comment:
Ok, I didn't rebased this branch but created PRs #1960 and #1959 to let CI check these changes separately and I will just rebase this PR so that only `OrganizationComponent` remains in the Files changed.